lua/nvim-tree/git/utils.lua

```diff
@@ -3,6 +3,22 @@ local log = require "nvim-tree.log"
 
 local has_cygpath = vim.fn.executable "cygpath" == 1
 
+local function norm_path(path)
+  -- git always returns path with forward slashes
+  if vim.fn.has "win32" == 1 then
```

Comment:
Windows... is always a problem. I cannot verify it however you will be testing it.

Please add `utils.lua` `is_msys2`

However... before you do that... I suspect the same problem may be exhibited on cygwin and WSL. Please test those two as well and maybe do this for `is_windows`